src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static int magic = 100;
```

Comment:
Read rules about constant. Bad naming - what `magic` is? 

```suggestion
    private static final int CONSTANT_NAME = 100;
```